### PR TITLE
Feature docker tailwindcss

### LINKED ISSUE #456: Run Tailwind CSS in Docker
### What's your idea?

Create a set-up for running Tailwind CSS in a Docker container

A work currently in progress, based on [tristanstraub/docker-tailwindcss](https://github.com/tristanstraub/docker-tailwindcss).

### Impact

Low

### Urgency

Now

### Code of Conduct

- [X] I agree to follow this project's Code of Conduct

### PR DESCRIPTION
**Pull request by giulio-giunta**
This adds configuration in the backend Dockerfile to install nodejs and npm in the django image and adds instructions in the readme.md to run tailwind commands

## Related Issue
resolves #456

## Motivation and Context
This change allows to run tailwind commands in docker-compose.

## How Has This Been Tested?
The commands successfully ran in docker-compose

## Types of changes

- [ ] Bug fix (non-breaking change which fixes an issue)
- [x] New feature (non-breaking change which adds functionality)
- [ ] Breaking change (fix or feature that would cause existing functionality to change)

## Checklist:

- [x] My code follows the code style of this project.
- [x] My change requires a change to the documentation.
- [x] I have updated the documentation accordingly.
- [x] I have read the **[CONTRIBUTING](CONTRIBUTING.md)** document.
- [ ] I have added tests to cover my changes.
- [ ] All new and existing tests passed.

### CODE REVIEW

File: k8s/test-environment/deploy-test.yaml [hunk 1]
```diff
@@ -0,0 +1,196 @@
...
+  SAND_PRICE_ID: "price_1ImwyKJh8KDe9GPFrw4mWFai"
+  DONATION_PRODUCT_ID: "prod_HnmOmrwV7zDBfw"
+  GMAIL_APP_PASSWORD: "$GMAIL_APP_PASSWORD"
+  DATABASE_HOST: "postgres"
+  DATABASE_PORT: "5432"
+  DATABASE_NAME: "geekzone"
+  DATABASE_USER: "${TEST_POSTGRES_USER}"      
+  DATABASE_PASSWORD: "${TEST_POSTGRES_PASSWORD}"
+  CELERY_BROKER_URL: "${CELERY_BROKER_URL}"
+  #CELERY_BACKEND: "amqp://h3_ZfqkTGiGmQIU5U6Y5YMsC-hKz6CyL:aWP_KIM-ZIZFNmixt8GBbnigKM5oWPFb@172.20.159.27"
```
Comment (jamesgeddes) [inline k8s/test-environment/deploy-test.yaml:27]: Do we need this comment? We should avoided commented code in master

File: k8s/test-environment/deploy-test.yaml [hunk 2]
```diff
@@ -0,0 +1,196 @@
...
+
+---
+apiVersion: v1
+kind: Service
+metadata:
+  name: test-gz-web
+  namespace: ${PR_NUMBER}-${CIRCLE_PROJECT_REPONAME}
+  annotations:
+    external-dns.alpha.kubernetes.io/hostname: ${PR_NUMBER}-${CIRCLE_PROJECT_REPONAME}-test.geek.zone
+#    external-dns.alpha.kubernetes.io/ttl: "120" #optional
```
Comment (jamesgeddes) [inline k8s/test-environment/deploy-test.yaml:116]: Can we avoid commented code in master please?

File: k8s/test-environment/deploy-test.yaml [hunk 3]
```diff
@@ -0,0 +1,196 @@
...
+---
+apiVersion: networking.k8s.io/v1beta1
+kind: Ingress
+metadata:
+  name: test-ingress
+  namespace: ${PR_NUMBER}-${CIRCLE_PROJECT_REPONAME}
+  annotations:
+    nginx.ingress.kubernetes.io/app-root: /memberships/register 
+#    nginx.ingress.kubernetes.io/default-backend: nginx-custom-default-backend 
+#    nginx.ingress.kubernetes.io/custom-http-errors: "404,415"
```
Comment (jamesgeddes) [inline k8s/test-environment/deploy-test.yaml:185]: Another set of commented code, please avoid

File: sonar-project.properties [hunk 4]
```diff
@@ -0,0 +1,16 @@
+# must be unique in a given SonarQube instance
+#sonar.projectKey=my:project
```
Comment (jamesgeddes) [inline sonar-project.properties:2]: Commented code

File: sonar_run.sh [hunk 5]
```diff
@@ -0,0 +1,35 @@
+#!/bin/bash 
+
+docker-compose up -d 
+sudo snap install jq
+# sudo apt-get install jq -y
```
Comment (jamesgeddes) [inline sonar_run.sh:5]: Commented code

**Review by jamesgeddes — CHANGES_REQUESTED** [review]

File: k8s/test-environment/deploy-test.yaml (hunk 1 above)
Comment (giulio-giunta) [reply k8s/test-environment/deploy-test.yaml:27]: I'll delete this PR and create a new one

File: k8s/test-environment/deploy-test.yaml (hunk 2 above)
Comment (giulio-giunta) [reply k8s/test-environment/deploy-test.yaml:116]: I'll delete this PR and create a new one

File: k8s/test-environment/deploy-test.yaml (hunk 3 above)
Comment (giulio-giunta) [reply k8s/test-environment/deploy-test.yaml:185]: I'll delete this PR and create a new one

File: sonar-project.properties (hunk 4 above)
Comment (giulio-giunta) [reply sonar-project.properties:2]: I'll delete this PR and create a new one

**Comment by giulio-giunta** [comment]
There are files that are not meant to be in this branch, therefore I will close this PR and create a new one with only the needed changes

**Pull request closed without merging**